hilti/include/base/util.h

```diff
@@ -408,6 +408,15 @@ using hilti::rt::escapeBytes;
 using hilti::rt::escapeUTF8;
 using hilti::rt::expandEscapes;
 
+/**
+ * Wrapper for `escapeBytes` that produces a valid C++ string literal.
+ *
+ * @param str string to escape
```

Comment:
```suggestion
 * @param s string to escape
```

Btw, there's a clang warning for errors like that, #240.